packages/checkout-common/icon.ts

```diff
@@ -0,0 +1,4 @@
+import { SvgIconTypeMap } from "@material-ui/core";
+import { OverridableComponent } from "@material-ui/core/OverridableComponent";
+
+export type IconComponent = OverridableComponent<SvgIconTypeMap<{}, "svg">>;
```

Comment:
I don't think types from material should be in common - Material itself is only used in `saleor-app-checkout`

Reply:
This type is used in `checkout` as well. I don't know why. I don't know how. We should get rid of material-ui anyway 🙃